models/curvefi/ethereum/curvefi_ethereum_view_pools.sql

```diff
@@ -292,6 +260,8 @@ SELECT
     undercoin1,
     undercoin2,
     undercoin3,
+    array(coin0, coin1, coin2, coin3) as coins,
+    array(undercoin0, undercoin1, undercoin2, undercoin3) as undercoins,
```

Comment:
@jeff-dude here's where the missing columns are added, but the CI isn't running this model first so the old version of this model referenced in trades is missing the column.